common/changes/@binaris/shift-db/query-language_2019-07-09-09-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@binaris/shift-db",
+      "comment": "Implement query language",
+      "type": "major"
```

Comment:
This gives version 1.0.0 which opts-in into semver, until now we tried to only use patch versions, do you think we should switch?